packages/commercetools/theme/pages/Checkout/Payment.vue

```diff
@@ -0,0 +1,383 @@
+          <SfInput
+            :value="billingDetails.streetNumber"
+            @input="streetNumber => setBillingDetails({ streetNumber })"
+            label="House/Apartment number"
+            name="apartment"
+            class="form__element form__element--half form__element--half-even"
+            required
+            :valid="!errors[0]"
+            :errorMessage="errors[0]"
+          />
+        </ValidationProvider>
+        <ValidationProvider name="city" rules="required|min:2" v-slot="{ errors }" slim>
+          <SfInput
+            :value="billingDetails.city"
+            @input="city => setBillingDetails({ city })"
+            label="City"
+            name="city"
+            class="form__element form__element--half"
+            required
+            :valid="!errors[0]"
+            :errorMessage="errors[0]"
+          />
+        </ValidationProvider>
+        <ValidationProvider name="zipCode" rules="required|min:2" v-slot="{ errors }" slim>
+          <SfInput
+            :value="billingDetails.postalCode"
+            @input="postalCode => setBillingDetails({ postalCode })"
+            label="Zip-code"
+            name="zipCode"
+            class="form__element form__element--half form__element--half-even"
+            required
+            :valid="!errors[0]"
+            :errorMessage="errors[0]"
+          />
+        </ValidationProvider>
+        <ValidationProvider name="country" rules="required|min:2" v-slot="{ errors }" slim>
+          <SfSelect
+            :selected="billingDetails.country"
+            @change="country => setBillingDetails({ country })"
+            label="Country"
+            class="form__element form__element--half form__select sf-select--underlined"
+            required
+            :valid="!errors[0]"
+            :errorMessage="errors[0]"
+          >
+            <SfSelectOption
+              v-for="countryOption in COUNTRIES"
+              :key="countryOption.key"
+              :value="countryOption.key"
+            >
+              {{ countryOption.label }}
+            </SfSelectOption>
+          </SfSelect>
+        </ValidationProvider>
+        <ValidationProvider name="phone" rules="required|min:2" v-slot="{ errors }" slim>
+          <SfInput
+            :value="billingDetails.contactInfo.phone"
+            @input="phone => setBillingDetails({ contactInfo: { phone } })"
+            label="Phone number"
+            name="phone"
+            class="form__element form__element--half form__element--half-even"
+            required
+            :valid="!errors[0]"
+            :errorMessage="errors[0]"
+          />
+        </ValidationProvider>
+      </div>
+    <SfHeading
+      :level="3"
+      title="Payment methods"
+      class="sf-heading--left sf-heading--no-underline title"
+    />
+    <div class="form">
+      <div class="form__element payment-methods">
+        <SfRadio
+          v-for="item in paymentMethods"
```

Comment:
What do you think to add `data-cy` to interactive elements so QA team can handle E2E?